test/lit.site.cfg.in

```diff
@@ -88,6 +88,11 @@ config.available_features.add("CMAKE_GENERATOR=@CMAKE_GENERATOR@")
 if "@SWIFT_ENABLE_SOURCEKIT_TESTS@" == "TRUE":
     config.available_features.add('sourcekit')
 
+if "@SWIFT_ENABLE_LLD_LINKER@" == "TRUE":
+    config.android_linker_name = "lld"
+else:  # even if SWIFT_ENABLE_GOLD_LINKER isn't set, we've to use a modern ld.
```

Comment:
It isn't about a modern ld.  BFD does not handle the protected relocations correctly and generates COPY relocs which android cannot handle.

Reply:
I will change the comments here and below.